src/store/pnp.js

```diff
@@ -531,8 +537,8 @@ const actions = {
    * them or let them connect to you.
    */
   async [CHANGE_REMOTE_PEER_ID] ({ state, commit, dispatch }, remotePeerId) {
-    commit(NEW_REMOTE_PEER_ID, remotePeerId)
     commit(PEER_DISCONNECTED)
```

Comment:
@vickywane good catch.

Reply:
@vickywane what UI flow was compromised due to this issue? Can you add it as a test case?